src/h2olog/misc/gen_raw_tracer.py

```diff
@@ -423,13 +434,29 @@ def build_tracer(context, metadata):
 #endif
 """
 
-  c += r"""
+  if fully_specified_probe_name == "h2o:socket_lookup_flags":
+    c += r"""
+#ifdef H2OLOG_SAMPLING_RATE
+  uint64_t flags = event.socket_lookup_flags.original_flags;
+  int skip_tracing = bpf_get_prandom_u32() > (H2OLOG_SAMPLING_RATE * U32_MAX);
+  if (skip_tracing) {
+    flags |= H2O_EBPF_FLAGS_SKIP_TRACING_BIT;
+  }
+  int64_t ret = h2o_return.insert(&event.socket_lookup_flags.tid, &flags);
+  if (ret != 0)
+    bpf_trace_printk("failed to insert 0x%%lx in %s with errno=%%ld\n", flags, -ret);
```

Comment:
https://www.kernel.org/doc/Documentation/printk-formats.txt

This document says that format specifiers for 64bit data type are `%lld` or `%llu` or `%llx`.